CameraXBasic/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -53,10 +55,11 @@
     <fragment
         android:id="@+id/gallery_fragment"
         android:name="com.android.example.cameraxbasic.fragments.GalleryFragment"
+        tools:layout="@layout/fragment_gallery"
```

Comment:
The fixes the navigation graph preview in Android Studio.

Reply:
This seems like a bad idea, since the layout is loaded dynamically. If the layout is changed, this will be out of date and that will be very confusing. I suggest reverting this change.